connectivity/tests/pod.go

```diff
@@ -59,6 +59,9 @@ func (s *podToPod) Run(ctx context.Context, t *check.Test) {
 
 					a.ValidateFlows(ctx, client, a.GetEgressRequirements(check.FlowParameters{}))
 					a.ValidateFlows(ctx, echo, a.GetIngressRequirements(check.FlowParameters{}))
+
+					a.ValidateMetrics(ctx, echo, a.GetIngressMetricsRequirements())
+					a.ValidateMetrics(ctx, echo, a.GetEgressMetricsRequirements())
```

Comment:
We probably want to add this to all tests where it makes sense (check for calls to `a.ValidateFlows` on where to add)

Reply:
I am pretty unsure of what I've done here, so please be careful in the review :)